code/modules/mob/living/simple_animal/guardian/types/standard.dm

```diff
@@ -2,6 +2,7 @@
 /mob/living/simple_animal/hostile/guardian/punch
 	melee_damage_lower = 20
 	melee_damage_upper = 20
+	next_move_modifier = 0.9 //attacks 20% faster
```

Comment:
You are correct that 0.5 is double, that's 50% faster. see? 0.5 -> 50% (or "double") faster, because you have 50% less delays, going from 1 to 0.9 is 10% less delays, so 10% faster.